components/Head/Head.tsx

```diff
@@ -55,6 +55,7 @@ const Head = ({
 
   return (
     <NextHead>
+      <html lang="en" />
```

Comment:
It will not work this way unfortunately. `<html>` is not a auto-closing tag, it should wrap all of the page content. E. g.:

```
<html>
  <head>...</head>
  <body>...</body>
</html>
```

And we already have one html tag on the page wrapping everything. To add `lang` attribute to it we need to do following:

1. Add `pages/_document.tsx` page.
2. Add the following content to it https://nextjs.org/docs/advanced-features/custom-document
3. Add `lang` to `Html` tag there.

Reply:
I was trying to use next.js docs similar to the blog but found we had a blend in docs. This solution was other a few experiments.. and it does work.

<img width="548" alt="image" src="https://user-images.githubusercontent.com/559288/202480577-9e7b9464-4927-4b75-922d-a56412314090.png">


( I know it might not look right.. but if it works 🤷 ), I can also open a PR if you think that's the best practice.

Reply:
Checked it locally. And it work incorrectly unfortunately. :(

Console is throwing errors:

<img width="764" alt="image" src="https://user-images.githubusercontent.com/451207/202497595-a1827de9-08cf-4fd2-90ad-e663fc1e5628.png">

And this is how DOM looks like now:

<img width="526" alt="image" src="https://user-images.githubusercontent.com/451207/202497499-5e36c92f-6ad5-4e0a-86e0-752ccfaea281.png">

See that `<title>` and `<link>` tags are actually in the `<body>` and not in the `<head>`. My guess is that browser meets the DOM structure which shouldn't exist (`<html>` inside `<head>`) and tries to autofix it by discarding code before `html`, inserting default head content and adding opening `<body>` tag before next lines.

Also preview build trowing exceptions for me and stops working in FF.

I think moving it to `_document` will be safer.